src/dialogs/prompt-dialog.js

```diff
@@ -97,71 +92,61 @@ class PromptDialog extends Dialog {
   }
 
   /**
-   * @param {Array.<Object>} candidates -
-   * array of raw entities given by the extractor. They are candidates for the entity parameters
+   * Computes matched and missing entities.
+   * @param {Array.<Object[]>} candidates - array of raw entities given by the extractor.
+   * They are candidates for the entity parameters
    * @param {Object} parameters - map of entities expected by the dialog: {
    *   <entityName>: {
    *     dim: String,
-   *     priority: Number or Function,
+   *     priority: Number,
    *     isFulfilled: Function()
    *     reducer: Function(),
    *   }
    * }
    * @param {Object} dialogEntities - a map of the entities already matched for this dialog: {
    *   <entityName>: <messageEntity>
    * }
-   * @returns {Object} object containing missingEntities(subset of expectedEntities) and
-   * matchedEntities (same structure as dialogEntities)
+   * @returns {Object} object containing missingEntities and matchedEntities
+   * (same structure as dialogEntities)
    */
   computeEntities(candidates, parameters, dialogEntities = {}) {
     logger.debug('computeEntities', { candidates, parameters, dialogEntities });
     // Setup default values for entities
     const dialogParameters = Object.keys(parameters).reduce((allEntities, name) => {
       const dialogParameter = parameters[name];
-      // If the fulfilled function is not defined, we consider that the fulfilled condition
-      // is met if the entity simply exists (not null or undefined)
-      const isFulfilled = dialogParameter.isFulfilled || (entity => entity != null);
-
       return {
         ...allEntities,
         [name]: {
           dim: dialogParameter.dim,
           // If the reducer function is not defined, we replace the old entities by the new ones
           reducer: dialogParameter.reducer || ((oldEntities, newEntities) => newEntities),
-          isFulfilled,
+          // If the isFulfilled function is not defined, we check that the entity is not null
```

Comment:
`!= null` (with only one `=`) checks for null and undefined, not only null (and is pretty common)